deploy/007_DeployNodes.ts

```diff
@@ -22,9 +24,18 @@ async function main() {
     if (node?.prices?.[0] !== undefined) node.prices[0] = parseUSDC(node.prices[0]).toString();
     if (node?.prices?.[1] !== undefined) node.prices[1] = parseUSDC(node.prices[1]).toString();
   }
+
+  // import nodes data
   const importArgs = [nodesData, true] as const;
   if (confirm(hre, `Execute EarthfastNodes.unsafeImportData ${stringify(importArgs)}`)) {
-    await wait(nodes.unsafeImportData(...importArgs));
+    // Split the nodes array into smaller chunks
+    for (let i = 0; i < nodesData.length; i += BATCH_SIZE) {
+      const batch = nodesData.slice(i, i + BATCH_SIZE);
+
+      console.log(`Importing nodes batch ${i / BATCH_SIZE + 1} (${batch.length} nodes)`);
+
+      await wait(nodes.unsafeImportData(batch, i + BATCH_SIZE >= nodesData.length));
```

Comment:
data import was failing when i attempted a full redeployment as the number of extant nodes was higher than the contract could process